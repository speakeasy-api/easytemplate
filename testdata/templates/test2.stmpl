{{.Global.Test}}
{{.Local.Added}}
```sjs
var local = context.Local;
context.Local = { Added: local };
render("js output");
render("another line");

var templated = templateString("templates/test3.stmpl", {Test: "from test2.stmpl sjs block"});
render(templated);
sjs```