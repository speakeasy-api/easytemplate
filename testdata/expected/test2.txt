global
from test.stmpl
js output
another line
from test2.stmpl sjs block